universal-login-react/src/ui/commons/ExplorerLinkOrMessage.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {getEtherscanUrl} from '../../core/utils/getEtherscanUrl';
+
+interface ExplorerLinkOrMessageProps {
+  chainName: string;
+  transactionHash?: string;
+  message?: string;
+  error?: string;
+}
+
+export const ExplorerLinkOrMessage = ({chainName, transactionHash, message, error} : ExplorerLinkOrMessageProps) => (
```

Comment:
Let's split that PR into two parts:
- move ExplorerLinkOrMessage to react
- rest

btw `ExplorerLinkOrMessage` makes me suffer, could really be just Explorer

Reply:
Can we btw refactor `ExplorerLinkOrMessage`?

Let just make it:
```ts
<ExplorerLink chainName='mainnet' target='0x...' />
```

And remove message/error.